notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/action/CompleteOnRedirectAction.java

```diff
@@ -36,14 +35,16 @@
  *
  * @author mglazier
  */
-public class StateChangeAction extends NotificationAction {
+public class CompleteOnRedirectAction extends NotificationAction {
 
-    public StateChangeAction() {
+    private static final long serialVersionUID = 1L;
+
+    public CompleteOnRedirectAction() {
         // Set a default label;  most use cases will use the setter and override
         setLabel("SET-STATE");
```

Comment:
Good point!  (Though I have a feeling the label is ultimately being overridden -- in the JPA service -- by the value of the column of the same name.)